### ISSUE TITLE
When send to an invalid Bjj, Continue button is blocked without any error message

### ISSUE THREAD

**Issue by invocamanman** [hermeznetwork/wallet-ui#594]
## Summary of Bug
When send to an invalid Bjj, `Continue` button is blocked without any error message.
Like ethereum address it should print some error message like: `the babyjub introduced is invalid`

### Steps to Reproduce
Go to send
Put any amount
In the address put an invalid bjj like: `hez:testtesttesttesttesttesttesttesttesttesttest`

### Browser information
- Browser: `Chrome`
- Device/OS: `Ubuntu`
### Additional Information & Screenshots:

Every time Continue button is clicked the console show the following message:
```
xhr.js:177 GET https://api.testnet.hermez.io/v1/accounts?BJJ=hez:testtesttesttesttesttesttesttesttesttesttest&tokenIds=3&order=ASC&limit=20 400

createError.js:16 Uncaught (in promise) Error: Request failed with status code 400
    at e.exports (createError.js:16)
    at e.exports (settle.js:17)
    at XMLHttpRequest.h.onreadystatechange (xhr.js:62)
```

**Comment by rauljareno** [comment]
happening too in native wallet, need to fix also

**Comment by AlbertoElias** [comment]
Closed by #627
